docs/usage/examples.md

```diff
@@ -0,0 +1,124 @@
+# SQL Examples to Analyze Your Data
+
+_If you want to know what the database schema looks like, you can refer to the [diagram about gitbase entities and relations](https://docs.sourced.tech/gitbase/using-gitbase/schema#database-diagram), or just use regular `SHOW` or `DESCRIBE` queries._
+
+_In gitbase repository, you will find more [SQL examples of queries](https://docs.sourced.tech/gitbase/using-gitbase/examples)._
+
+
+## Index
+
+* [Queries For Repositories](#queries-for-repositories)
+* [Queries With Files](#queries-with-files)
+* [Queries With UASTs](#queries-with-uasts)
+* [Queries About Comitters](#queries-about-comitters)
+
+
+## Queries For Repositories
+
+**Show me the repositories I am analyzing:**
+
+```sql
+SELECT * FROM repositories;
+```
+
+**Last commit messages for HEAD for every repository**
+
+```sql
+SELECT commit_message
+FROM refs
+NATURAL JOIN commits
+WHERE ref_name = 'HEAD';
+```
+
+**Top 10 repositories by commit count from [HEAD](https://git-scm.com/book/en/v2/Git-Internals-Git-References#ref_the_ref):**
+
+```sql
+SELECT repository_id,commit_count
+FROM (
+    SELECT
+        repository_id,
+        COUNT(*) AS commit_count
+    FROM ref_commits
+    WHERE ref_name = 'HEAD'
+    GROUP BY repository_id
+) AS q
+ORDER BY commit_count DESC
+LIMIT 10;
+```
+
+**10 top repos by file count at HEAD**
+
+```sql
+SELECT repository_id, num_files FROM (
+    SELECT COUNT(f.*) num_files, f.repository_id
+    FROM ref_commits r
+    NATURAL JOIN commit_files cf
+    NATURAL JOIN files f
+    WHERE r.ref_name = 'HEAD'
+    GROUP BY f.repository_id
+) AS t
+ORDER BY num_files DESC
+LIMIT 10;
+```
+
+
+## Queries With Files
+
+**Query for all LICENSE & README files across history:**
+
+```sql
+SELECT file_path, repository_id, blob_size
+FROM files
+WHERE
+    file_path = 'LICENSE'
+    OR file_path = 'README.md';
+```
+
+**Query all files at [HEAD](https://git-scm.com/book/en/v2/Git-Internals-Git-References#ref_the_ref):**
+
+```sql
+SELECT cf.file_path, f.blob_size
+FROM ref_commits rc
+NATURAL JOIN commit_files cf
+NATURAL JOIN files f
+WHERE
+    rc.ref_name = 'HEAD'
+    AND rc.history_index = 0;
+```
+
+
+## Queries With UASTs
+
+_**Note**: UAST values are returned as binary blobs; they're best visualized in the `web sql` interface rather than the CLI where are seen as binary data._
```

Comment:
`web sql` was the command in engine!
```suggestion
_**Note**: UAST values are returned as binary blobs; they're best visualized in the web UI rather than the CLI where are seen as binary data._
```